bootstrap.sh

```diff
@@ -318,14 +315,16 @@ add_dependency TENSORFLOW_ENABLED "tensorflow"
 add_disabled_option OPC_ENABLED ${FALSE} "ENABLE_OPC"
 add_dependency OPC_ENABLED "mbedtls"
 
+JNI_ENABLED=${FALSE}
+add_dependency JNI_ENABLED "jnibuild"
```

Comment:
it seems to me, that JNI_ENABLED no longer being added to OPTIONS affects the platform specific dependency installer scripts, specifically they iterate over the OPTIONS and then install their dependencies, could this cause that java will no longer be installed even if we enable jni?

Reply:
Yes, you're right.  I have reverted the JNI part of the change.